search_knn_test.go

```diff
@@ -131,13 +131,15 @@ func TestSimilaritySearchPartitionedIndex(t *testing.T) {
 	}
 
 	index := NewIndexAlias()
+	var reqSort = search.SortOrder{&search.SortScore{Desc: true}, &search.SortDocID{Desc: true}, &search.SortField{Desc: false, Field: "content"}}
 	for testCaseNum, testCase := range testCases {
+		originalRequest := searchRequests[testCase.queryIndex]
 		for _, operator := range knnOperators {
+			// ------------------------------------------------------------------------------------------------------------------
```

Comment:
Drop these commented out lines here and below. If you're trying to use as a separator between code blocks, just use an empty line instead.